sdk/core/aborter/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@azure/core-aborter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/aborter.js",
+  "module": "./dist-esm/aborter.js",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
+    "build:test": "rollup -c rollup.test.config.js 2>&1",
+    "build": "npm run build:es6 && npm run build:nodebrowser",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm dist-test types temp browser/*.js* browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "lint": "true",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "pretest": "npm run build:test",
+    "test:browser": "echo \"Browser tests not yet supported\"",
+    "test:node": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" mocha --require ts-node/register --require source-map-support/register --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json --full-trace --no-timeouts test/*.spec.ts",
+    "test": "npm run test:node && npm run test:browser",
+    "unit-browser": "npm run build && npm run test:browser",
+    "unit-node": "npm run build && npm run test:node"
+  },
+  "types": "./types",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
I guess this is from `npm init` and we use `MIT`?

Reply:
good catch, also need LICENSE file.

Reply:
Fixed